crates/phala-types/src/contract.rs

```diff
@@ -201,7 +200,6 @@ pub enum ClusterPermission<AccountId> {
 pub struct ClusterInfo<AccountId> {
     pub owner: AccountId,
     pub permission: ClusterPermission<AccountId>,
-    pub workers: Vec<WorkerPublicKey>,
```

Comment:
It breaks the old SDK. Given we already have devphase and Swanky integrated, I suggest to change it to an empty field instead of removing it.

Reply:
Good point. Added it back.

Reply:
As devphase doesn't rely on this field, I think removing it would be better than keep an empty field there, as the empty field might be confusing and misleading for people.

Reply:
I'm not sure if the older JS SDK would fail to decode it. Theoretically pjs should load the correct metadata from the node, and decode it with the correct type definition. However if it can't, other tools like devphase would be broken by this change.

Reply:
There is no bundled metadata for the on-chain types in the JS SDK, so it there should be no difference between old and new version of JS-SDK on decoding.
What matters is that old JS-SDK or frontend page might use the field to do something.